### ISSUE TITLE
Sentinel R.1 ASTOR [Global Express] - United States (Army), 2022, 5x

### ISSUE THREAD

**Issue by tbrandon517** [PygmalionOfCyprus/cmo-db-requests#676]
### DB Selector

DB3K

### DBID to Copy

aircraft/2707/DB3K

### Hypothetical

Yes

### Name

Sentinel R.1 ASTOR [Global Express]

### Country

United States

### Service

Army

### In Commission

2022-

### Sensors

None Changes Known

### Mounts

None Changes Known

### Loadout Options

None Changes Known

### Other Changes

The retired Sentinel R1 fleet of five aircraft was sold to a US consortium of three companies– Springfield Air, Raytheon US, and Bombardier, with many suspecting the U.S. Army as the ultimate end user/customer.

### Sources

https://www.thedrive.com/the-war-zone/43184/unwanted-british-sentinel-radar-planes-to-be-picked-up-by-us-army-report

https://ukdefencejournal.org.uk/british-sentinel-aircraft-to-be-sold-to-u-s-army-instead-of-being-scrapped/

https://www.janes.com/defence-news/news-detail/uk-confirms-sentinel-sale-to-us

**Comment by tbrandon517** [comment]
This We'll Defend

**Comment by PygmalionOfCyprus** [comment]
It may have been rumored to go to the US Army but I think it's pretty clear this never actually happened. We have the "real" replacement platforms in the DB so I don't think we need to offer this as a hypo capability: it's not really a big "what-if," just a different airframe that probably would have ended up getting much of the same equipment as the "real" ISR platforms got.

**Comment by tbrandon517** [comment]
> It may have been rumored to go to the US Army but I think it's pretty clear this never actually happened. We have the "real" replacement platforms in the DB so I don't think we need to offer this as a hypo capability: it's not really a big "what-if," just a different airframe that probably would have ended up getting much of the same equipment as the "real" ISR platforms got.

Yes it doesn't seem they really did anything with the airframe that I can find in public sources. If they weren't scrapped they may have just been used for development support and reference.
